<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>490</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>470</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="canvasPlaceholder" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>470</width>
      <height>392</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>390</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498db;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>172</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QComboBox" name="yAxisComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="xAxisLabel">
       <property name="text">
        <string>X-Axis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="yAxisLabel">
       <property name="text">
        <string>Y-Axis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="xAxisComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="plotButton">
       <property name="text">
        <string>Plot Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveFilesButton">
       <property name="text">
        <string>Save Files</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
